_monitoring-plugins/alerting/monitors.md

```diff
@@ -43,75 +43,7 @@ Trigger | Conditions that, if met, generate *alerts*.
 Tag | A label that can be applied to multiple queries to combine them with the logical OR operation in a per document monitor. You can't use tags with other monitor types.
```

Comment:
I don't mind "can't," as it establishes a more casual tone. "Cannot" might be a bit formal here.

Reply:
Thanks, Nate. "Can't" is the only contraction I have a problem with, since the term "cannot" is a little more friendly and more commonly used. Would be great to standardize on it.